### PR TITLE
docs: modify 013.md, 013 CN version and README

### PR DESCRIPTION
**Pull request by JoannaWu84**

### CODE REVIEW

File: newsletters/013.cn.md [hunk 1]
```diff
@@ -2,10 +2,42 @@
...
+
+  ![pic19.jpg](./images/pic19.jpg)
+
+  
+
+  [相关新闻](https://weibo.com/ttarticle/p/show?id=2309404956447401443408)
+  
++ Milk-V Duo 发布了 1.0.5 版本。在此次固件升级中新增了小核支持。更多详情请参考下方链接
+
+  [Detailed Info](https://github.com/milkv-duo/duo-buildroot-sdk/releases/tag/Duo-V1.0.5)
```
Comment (XieJiSS) [inline newsletters/013.cn.md:38]: 这里改成中文？

File: newsletters/013.md [hunk 2]
```diff
@@ -2,10 +2,41 @@
 
 ## Editor's Note
 
-Welcome to the thirteenth issue of the SG2042 Newsletter.
+Welcome to the thirteenth issue of the SG2042 Newsletter. We are thrilled to announce that the Shandong University-SOPHGO RISC-V Research Institute has been officially established. This is an important milestone that will greatly promote the promotion and application of RISC-V processors in China and provide strong technical support for industry development.
```
Comment (XieJiSS) [inline newsletters/013.md:5]: promote the promotion 这个表述最好 rephrase 下~

File: newsletters/013.md [hunk 3]
```diff
@@ -2,10 +2,41 @@
...
 
 ## Highlights
 
++ On October 13, 2023, the Shandong University-SOPHGO RISC-V Research Institute was officially unveiled. This marks the establishment of the first new research institute in Chinese universities with RISC-V as its main research direction. The establishment of Shandong University-SOPHGO RISC-V Research Institute is not only a vibrant practice of combining industry, academia and research, but also demonstrates foresight, strategic significance and exemplar.
```
Comment (XieJiSS) [inline newsletters/013.md:9]: 这里有比较多的重复表述，英语行文上一般不用这么重复，比如说如果 unveiled -> established 然后就会发现变成了 was officially established... marks the establishment... 其实意思是重复的。the first new research center 也是一样的道理。可以稍作精简，意思传达到即可，比如 was officially announced, which is the first research institute...

File: newsletters/013.md [hunk 4]
```diff
@@ -2,10 +2,41 @@
...
+
+  RISC-V, as an open-source instruction set architecture, has received increasing attention and application in recent years. It has the advantages of flexibility, customizability and openness, bringing more opportunities for chip design and embedded systems. The collaboration between Shandong University and SOPHGO will promote in-depth research and wide application of RISC-V technology in China, providing new opportunities for the development of related fields.
+
+  ![pic16.jpg](./images/pic16.jpg)
+
+  
+
+  [Related news](https://mp.weixin.qq.com/s/0_2N6WI_Z2NRJ09joynn-w)
+
++ On October 11-12, 2023, the 5th Big Data & AI World was held at Marina Bay Sands in Singapore. SOPHGO was invited to appear at the show to demonstrate the technical strength of China's AI computing power, which attracted the attention of attendees from all over the world.
```
Comment (XieJiSS) [inline newsletters/013.md:19]: 最好是 & AI World event

File: newsletters/013.md [hunk 5]
```diff
@@ -2,10 +2,41 @@
...
+
+  
+
+  ![pic19.jpg](./images/pic19.jpg)
+
+  
+
+  [Related news](https://weibo.com/ttarticle/p/show?id=2309404956447401443408)
+  
++ Milk-V Duo has released the 1.0.5 firmware version. In this firmware update, it has released the generic freertos code for the second core (little one). For details, please refer to the link below.
```
Comment (XieJiSS) [inline newsletters/013.md:35]: it 指代不明确了。Generic freeRTOS codes for the second core (the little one) have been released in this version.

File: newsletters/013.md [hunk 6]
```diff
@@ -14,7 +45,9 @@ Most of the code is already open-source and can be obtained from repositories su
 
 + Sophgo Community work: https://github.com/sophgo/linux-riscv
 
-  + 
+  + Fix the bug in memblock
```
Comment (XieJiSS) [inline newsletters/013.md:48]: a bug

**Review by XieJiSS — CHANGES_REQUESTED** [review]

File: newsletters/013.cn.md (hunk 1 above)
Comment (JoannaWu84) [reply newsletters/013.cn.md]: fixed

File: newsletters/013.md (hunk 3 above)
Comment (JoannaWu84) [reply newsletters/013.md]: fixed

File: newsletters/013.md (hunk 4 above)
Comment (JoannaWu84) [reply newsletters/013.md]: fixed

File: newsletters/013.md (hunk 6 above)
Comment (JoannaWu84) [reply newsletters/013.md]: fixed

File: newsletters/013.md (hunk 5 above)
Comment (JoannaWu84) [reply newsletters/013.md]: has been revised into> In this firmware update, generic freeRTOS codes for the second core (the little one) have been released.

File: newsletters/013.md (hunk 2 above)
Comment (JoannaWu84) [reply newsletters/013.md]: has been revised into >  will greatly facilitate the promotion and application

**Review by XieJiSS — APPROVED** [review]
LGTM, thanks!

**Pull request merged**